database/index.js

```diff
@@ -44,18 +46,17 @@ const Personnel = mongoose.model('Personnel', personnelSchema);
  * QUERIES *
  ***********/
 
-const getPersonnel = (movie) => {
-  return Personnel.find({movies: mongoose.Types.ObjectId(movie)}).exec();
+const getPersonnel = (id) => {
+  return Movie.find().where('personnel._id').equals(id).exec();
 }
 
-const getMovies = (person) => {
-  return Movie.find().exec();
+const getMovies = () => {
+  return Movie.find().populate('personnel._id').exec();
 }
 
 /* Export schemas for testing and seeding the database */
  module.exports.Movie = Movie;
```

Comment:
instead of exporting each of these in its own line you could instead export them all together as an object.